### ISSUE TITLE
Pester may be vulnerable to RCE due to a vulnerability in Sparkle

### ISSUE THREAD

**Issue by jstr** [nriley/Pester#21]
Apps that use Sparkle versions prior to 1.13.1 are vulnerable to RCE if they check for updates over HTTP. See https://github.com/sparkle-project/Sparkle/releases/tag/1.13.1 and  http://arstechnica.com/security/2016/02/huge-number-of-mac-apps-vulnerable-to-hijacking-and-a-fix-is-elusive/

According to my system the latest release of Pester uses Sparkle version 1.7.1:

```
find /Applications -path '*Autoupdate.app/Contents/Info.plist' -exec echo {} \; -exec grep -A1 CFBundleShortVersionString '{}' \; | grep -v CFBundleShortVersionString
[snipped]
/Applications/Pester.app/Contents/Frameworks/Sparkle.framework/Versions/A/Resources/Autoupdate.app/Contents/Info.plist
	<string>1.7.1</string>
```

And according to Little Snitch connects to sabi.net to check for updates on port 80. 

Therefore Pester may be vulnerable to RCE.

**Comment by nriley** [comment]
Thanks for the report.  I can't easily update Sparkle as the newer versions require a 64-bit runtime and ARC (as is, the version of Sparkle in Pester has some bits manually backported) but I can use SSL for the appcast.

Try this:

`defaults write net.sabi.Pester SUFeedURL https://sabi.net/nriley/software/Pester/updatez.xml`

That'll set up a testing appcast which will be accessed by SSL.  If it works for some other people I'll release it tomorrow.  Once 1.1b21 is released, you should go back to the original appcast with:

`defaults delete net.sabi.Pester SUFeedURL`

**Comment by nriley** [comment]
This should be fixed in 1.1b21. Please let me know if you experience any issues.
